doc/users/whats_new/2015-07-30_ensure_ax.rst

```diff
@@ -0,0 +1,18 @@
+Decorators to ensure an Axes exists & ``plt.quick_axes``
+--------------------------------------------------------
+
+Added a top-level function to `pyplot` to create a single axes
```

Comment:
Is that used internally or for eternal users of matplotlib, which want to do a new plotting function?